Port: Trans-Solar Federation

definitelynotspaghetti opened the pull request:
Порт солдатов и генерала ТСФ с оффов. Перенес всё их снаряжение к нам.
Добавил иконку для сб худа новую. Перенес шапки, одежду, рюкзаки и риги. В игре всё неплохо выглядит. Осталось подождать пока одобрят.
![solgov](https://user-images.githubusercontent.com/48985748/187693573-c36bec7c-6a31-4653-94e4-17209d03ee4f.png)

Commit:
Порт солговов с оффов к нам

Перенес одежду, риги и аутфиты

Commit:
Порт солговов с оффов к нам

Перенес одежду, риги и аутфиты

Commit:
Иконка для ТСФ + генерал солгова

Commit:
Иконка для ТСФ + генерал солгова

Commit:
Restricted species list update

Commit:
Restricted species list update

Commit:
Ещё иконки

Commit:
Ещё иконки

larentoun committed:
fix: vox_word flag for non-interrupted words (#1309)

Bizzonium committed:
Revert "Feat: add Robocop and Paladin lawsets to default pool (#733)"

This reverts commit e14f3a644052cccc1129555fc4c3bbfad52842e0.

BeebBeebBoob commented on code/modules/newscaster/obj/newscaster.dm:
```diff
@@ -80,6 +80,7 @@
 			/datum/job/chaplain,
 			/datum/job/ntnavyofficer,
 			/datum/job/ntspecops,
+			/datum/job/ntspecops/solgovspecops,
```
А я вот боюсь что будут проблемы везде где ты так сделал.
Не знаю под чистую скопировал смотреть как это подтип того что уже подтип страшна

```suggestion
			/datum/job/solgovspecops,
```

definitelynotspaghetti replied:
Это у оффов так. У них проблем не возникло, у нас тоже не должно

BeebBeebBoob reviewed (commented):
Предполагаю что делать по карте ничего не будешь? Ни базы своей вот у СолГов ни возможность админам спавниться из лобби за него.

larentoun committed:
fix: legion door opens on damage (#1307)

ref

Den109G committed:
Feat: add Espeon the pAIkemon (#953)

* pai-espeon

* icons for pai in hands

Den109G committed:
Feat: add Shitcurity Goblin smite (#964)

* Goblind-Updated

goblin

goblin for griefers

grammar problem

damage tweak

no1 saw it

no1 saw it 2

moved a dmi file to mob

Apply suggestion

topic up2date

Co-Authored-By: Vallat <vox.crit2013@gmail.com>

* new2Initialize

Co-authored-by: Vallat <vox.crit2013@gmail.com>

* blank-lines

Co-authored-by: Vallat <vox.crit2013@gmail.com>

Commit:
попытка исправить конфликт

Commit:
исправление конфликта

Commit:
Merge branch 'Trans-Solar-Federation' of https://github.com/frikadeliSpaghetti/Paradise into Trans-Solar-Federation

BeebBeebBoob commented on code/datums/ai_law_sets.dm:
```diff
@@ -83,7 +83,6 @@
 	name = "Робокоп"
 	law_header = "Основные директивы"
 	selectable = TRUE
-	default = TRUE
```
Не относится к ПР.

BeebBeebBoob commented on code/datums/ai_law_sets.dm:
```diff
@@ -96,7 +95,6 @@
 	name = "П.А.Л.А.Д.И.Н." //  // Р.Ы.Ц.А.Р.Ь. ("Ь." xD )
 	law_header = "Божественные Предписания"
 	selectable = TRUE
-	default = TRUE
```
Не относится к ПР.

BeebBeebBoob commented on code/game/turfs/simulated.dm:
```diff
@@ -103,13 +103,15 @@
 
 				if(TURF_WET_PERMAFROST) // Permafrost
 					M.slip("the frosted floor", 0, 5, tilesSlipped = 1, walkSafely = 0, slipAny = 1)
-	var/mob/living/simple_animal/hulk/Hulk = A
```
Не относится к ПР.

BeebBeebBoob commented on code/game/turfs/simulated.dm:
```diff
@@ -103,13 +103,15 @@
 	if (istype(A, /mob/living/simple_animal/hulk/clown_hulk))
 		if(!Hulk.lying)
 			playsound(src, pick('sound/effects/clownstep1.ogg','sound/effects/clownstep2.ogg'), CHANNEL_BUZZ)
+	if(istype(A, /mob/living/simple_animal/hostile/shitcur_goblin))
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -692,6 +692,7 @@ GLOBAL_LIST_INIT(admin_verbs_ticket, list(
 		ptypes += "Floor Cluwne"
 		ptypes += "Shamebrero"
 		ptypes += "Dust"
+		ptypes += "Shitcurity Goblin"
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -806,6 +807,11 @@ GLOBAL_LIST_INIT(admin_verbs_ticket, list(
 		if("Dust")
 			H.dust()
 			logmsg = "dust"
+		if("Shitcurity Goblin")
+			var/turf/T = get_turf(M)
+			var/mob/living/simple_animal/hostile/shitcur_goblin/goblin = new (T)
+			goblin.GiveTarget(M)
+			logmsg = "shitcurity goblin"
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/admin/topic.dm:
```diff
@@ -3009,95 +3009,75 @@
 				var/delete_mobs = alert("Clear all mobs?","Confirm","Yes","No","Cancel")
 				if(delete_mobs == "Cancel")
 					return
-
```
Не относится к ПР.
Да, потому что даже отступы могут быть полезны когда работают другие кодеры и это может вызвать у других конфликты. Вот твой ПР это про СолГов.

BeebBeebBoob commented on code/modules/mob/living/silicon/ai/say.dm:
```diff
@@ -143,7 +143,7 @@ GLOBAL_VAR_INIT(announcing_vox, 0) // Stores the time of the last announcement
 	var/i = 0
 	for(var/word in words)
-		addtimer(CALLBACK(GLOBAL_PROC, .proc/play_vox_word, word, src.z, null), i)
+		addtimer(CALLBACK(GLOBAL_PROC, .proc/play_vox_word, word, src.z, null), i, TIMER_CLIENT_TIME)
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/mob/living/silicon/pai/pai.dm:
```diff
@@ -26,7 +26,8 @@
 		"Box Bot" = "boxbot",
 		"Spider Bot" = "spiderbot",
-		"Fairy" = "fairy"
+		"Fairy" = "fairy",
+		"Espeon" = "pAIkemon_Espeon"
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm:
```diff
@@ -51,6 +51,14 @@ Difficulty: Medium
 	mouse_opacity = MOUSE_OPACITY_ICON
 	stat_attack = UNCONSCIOUS // Overriden from /tg/ - otherwise Legion starts chasing its minions
 
+/mob/living/simple_animal/hostile/megafauna/legion/adjustHealth(damage, updating_health)
+	. = ..()
+	if(!GLOB.necropolis_gate)
+		return
+	if(GLOB.necropolis_gate.legion_triggered)
+		return
+	GLOB.necropolis_gate.toggle_the_gate(src, TRUE)
+
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/mob/living/simple_animal/hostile/shitcur_goblin.dm:
```diff
@@ -0,0 +1,37 @@
+/mob/living/simple_animal/hostile/shitcur_goblin
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/ruins/objects_and_mobs/necropolis_gate.dm:
```diff
@@ -141,6 +141,7 @@ GLOBAL_DATUM(necropolis_gate, /obj/structure/necropolis_gate/legion_gate)
 /obj/structure/necropolis_gate/legion_gate
 	desc = "A tremendous, impossibly large gateway, set into a massive tower of stone."
 	sight_blocker_distance = 2
+	var/legion_triggered = FALSE
```
Не относится к ПР.

BeebBeebBoob commented on code/modules/ruins/objects_and_mobs/necropolis_gate.dm:
```diff
@@ -168,6 +169,7 @@ GLOBAL_DATUM(necropolis_gate, /obj/structure/necropolis_gate/legion_gate)
 /obj/structure/necropolis_gate/legion_gate/toggle_the_gate(mob/user, legion_damaged)
 	if(open)
 		return
+	GLOB.necropolis_gate.legion_triggered = TRUE
```
Не относится к ПР.

BeebBeebBoob reviewed (changes requested):
Также "Не относится к ПР." некоторые dmi изменения и .ogg

BeebBeebBoob commented:
icons/mob/shitkur_v6.dmi (0)
icons/obj/objects.dmi (0)
icons/mob/inhands/items_righthand.dmi (0)
icons/mob/head.dmi (0) (там ratge и другие)
icons/mob/inhands/items_lefthand.dmi (0)
icons/mob/pai.dmi (0)

definitelynotspaghetti commented:
Пересоздам лучше
